k8s-install/scripts/install_cni_test.go

```diff
@@ -246,3 +207,13 @@ var _ = Describe("install-cni.sh tests", func() {
 		})
 	})
 })
+
+func expectFilesEqual(filenameExpected, filenameActual string) {
+	expected, err := ioutil.ReadFile(filenameExpected)
+	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "failed to read expected file "+filenameExpected)
+	actual, err := ioutil.ReadFile(filenameActual)
+	ExpectWithOffset(1, err).NotTo(HaveOccurred(), "failed to read actual file "+filenameActual)
+
+	ExpectWithOffset(1, actual).To(Equal(expected), fmt.Sprintf(
+		"actual file (%s) differed from expected file (%s)", filenameActual, filenameExpected))
```

Comment:
Can you log out the content diff here?  It feels likely someone would want to see that.

Reply:
Gomega does that

Reply:
D'oh, of course.